src/features/nft/nftModals/reviewOfferModal.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import cn from 'classnames'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import { Button } from '../../../common/components/Buttons'
+import Link from '../../../common/components/Link'
+import { formatNumber } from '../../../common/utils/format'
+
+export type TOffer = {
+  id: number | string
+  user: string
+  price: number
+}
+
+export type TReviewOfferModal = {
+  title?: string
+  offers?: Array<TOffer>
+  isOpen: boolean
+  handleClose: () => void
+}
+
+const ReviewOfferModal = ({
+  title,
+  offers,
+  isOpen,
+  handleClose,
+}: TReviewOfferModal): JSX.Element => {
+  return (
+    <Modal isOpen={isOpen} handleClose={handleClose} size='3xl'>
+      <h2 className='mb-6'>Review an offer for “{title}”</h2>
+      <div className='mb-10'>
+        {offers?.map(({ id, user, price }, idx) => (
+          <div
+            key={id}
+            className={cn(
+              'flex flex-wrap align-center justify-between',
+              idx !== offers.length - 1 && 'mb-4',
+            )}
+          >
+            <h4 className='font-medium flex items-center'>
+              {formatNumber(price)} PSL &nbsp;{' '}
```

Comment:
Can we get the name(PSL) from `currencyName` because we have a testnet and the `PSL` will change to `PLS`.